.repo-metadata.json

```diff
@@ -2,14 +2,15 @@
     "name": "google-cloud-monitoring-metrics-scopes",
     "name_pretty": "Metrics Scopes API",
     "product_documentation": "https://cloud.google.com/monitoring/docs",
-    "client_documentation": "https://cloud.google.com/monitoring/settings/manage-api",
+    "client_documentation": "https://cloud.google.com/python/docs/reference/google-cloud-monitoring-metrics-scopes/latest",
     "issue_tracker": "https://issuetracker.google.com/savedsearches/559785",
-    "release_level": "ga",
+    "release_level": "stable",
     "language": "python",
     "library_type": "GAPIC_AUTO",
     "repo": "googleapis/python-monitoring-metrics-scopes",
     "distribution_name": "google-cloud-monitoring-metrics-scopes",
     "api_id": "monitoring.googleapis.com",
     "default_version": "v1",
-    "codeowner_team": ""
+    "codeowner_team": "",
+    "api_shortname": "google-cloud-monitoring-metrics-scopes"
```

Comment:
Should this be `monitoring`? to match the `api_id`?

Reply:
Fixed